rasa/core/channels/twilio_voice.py

```diff
@@ -229,13 +246,21 @@ def blueprint(
         async def health(request: Request) -> HTTPResponse:
             return response.json({"status": "ok"})
 
+        @twilio_voice_webhook.route("/recording", methods=["GET","POST"])
+        async def health(request: Request) -> HTTPResponse:
+            return response.json({"status": "ok","recording":"ok"})
+
         @twilio_voice_webhook.route("/webhook", methods=["POST"])
         async def receive(request: Request) -> HTTPResponse:
             sender_id = request.form.get("From")
             text = request.form.get("SpeechResult")
             input_channel = self.name()
             call_status = request.form.get("CallStatus")
-
+            call_sid=request.form.get('CallSid')
+            if call_sid not in call_sids:
+                call_sids.add(call_sid)
+                loop = asyncio.get_event_loop()
+                t1 = loop.create_task(self._twilio_record(call_sid))
```

Comment:
Did you consider using the `twilio` module here to record the call?

Reply:
No, I did not consider that. It made sense for me to record the voice call as I did.